.tool-versions

```diff
@@ -1 +1 @@
-nodejs 20.11.1
+nodejs 20.18.0
```

Comment:
I recommend we move to Node 22 if possible [since that's what the docs say we should be using](https://github.com/ithaka/pharos/blob/develop/docs/development/quick-start.md?plain=1#L21), and that we update the docs, maybe to remove reference to a version at all.

Reply:
Interesting. I was curious and it looks like [node 22 LTS support starts tomorrow](https://github.com/nodejs/Release). I can bump that and clean up the docs
<img width="1020" alt="image" src="https://github.com/user-attachments/assets/31fb5a65-b410-4028-8e7a-27e65c15faa8">